### PR TITLE
Add the timezone for PHP docker container through a parameter in docker-compose

### PR DESCRIPTION
**Pull request by DjayDev**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?|  no
| BC breaks?   | no
| Deprecations? |no
| Tests pass?   | yes|no
| Documentation | yes|no
| Translation   | yes|no
| Fixed tickets | 
| License       | MIT

Fix the #1726 issue.

Docker-compose should be upgraded to v1.6 to use the argument of the docker-compose.yml file.
In this sample, the argument is different from the default to show that it's the one from compose which is taken.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,42 +1,47 @@
...
+            - ./docker/logs/nginx:/var/log/nginx
+            - .:/var/www/html
+        links:
+            - php:php
+        command: nginx -c /nginx.conf
+    php:
+        build:
+            context: docker/php
+            args:
+                timezone: 'Europe/Monaco'
```
Comment (j0k3r) [inline docker-compose.yml]: Why did you put Monaco here and Paris in Dockerfile?

Comment (DjayDev) [inline docker-compose.yml]: I did it to show that the argument in the compose file is the one used.
The one in the dockerfile is a default one.
So like this, and as someone rightly commented, only the compose file need
to be modified.

I hope it's clear like that.

Comment (j0k3r) [inline docker-compose.yml]: It could be a good idea to drop a comment line on the composer file to explain that

**Comment by DjayDev** [comment]
I have added comments to explain it.
I hope it will be fine like that.

**Comment by j0k3r** [comment]
Excellent, thanks !
Could you rebase your branch against the v2 branch to avoid all these extra commits?

And we'll be ready to merge your changes :)

File: .gitignore [hunk 2]
```diff
@@ -38,3 +38,6 @@ docker/data/
 
 # To avoid crazy stuff on some PR, we must manually FORCE ADD IT on each new release
 composer.lock
+
+# To avoid IDE specific files
+.idea/*
```
Comment (nicosomb) [inline .gitignore]: Can you remove these lines please? 

It's better to put them in your global gitignore.

**Comment by FabienM** [comment]
:+1:

**Comment by DjayDev** [comment]
Thanks for the global gitignore. I was used to have one specific per
project in order to share also the coding conventions but it's better like
that.
I will fight with the rebase to submit a clean pull request.
Le 4 mars 2016 12:00, "Fabien Meurillon" <notifications@github.com> a
écrit :

**Comment by j0k3r** [comment]
I've fixed your commits in this PR https://github.com/wallabag/wallabag/pull/1771

**Pull request closed without merging**